src/javascript/SelectorTypes/DateTimePicker/DateTimePicker.jsx

```diff
@@ -12,15 +12,25 @@ const variantMapper = {
     DateTimePicker: 'datetime'
 };
 
+// Read global configuration
+const allowedOverridesDateFormat = ['MM/DD/YYYY', 'DD/MM/YYYY'];
+const forceDateFormat = window.contextJsParameters?.config?.contentEditor?.forceDateFormat;
+if (forceDateFormat && !allowedOverridesDateFormat.includes(forceDateFormat)) {
+    console.warn(`forceDateFormat as been set to an invalid value (${forceDateFormat}). Please use one of the following values: ${allowedOverridesDateFormat.join(', ')}`);
+}
+
+const useConfigDateFormatOverride = allowedOverridesDateFormat.includes(forceDateFormat) && forceDateFormat;
+
```

Comment:
Code looks ok but would prefer to wrap it in a function and call this to set `useConfigDateFormatOverride` variable.

```suggestion
// Read global configuration
function getDateFormatOverride() {
    const allowedOverridesDateFormat = ['MM/DD/YYYY', 'DD/MM/YYYY'];
    const forceDateFormat = window.contextJsParameters?.config?.contentEditor?.forceDateFormat;
    if (forceDateFormat && !allowedOverridesDateFormat.includes(forceDateFormat)) {
        console.warn(`forceDateFormat as been set to an invalid value (${forceDateFormat}). Please use one of the following values: ${allowedOverridesDateFormat.join(', ')}`);
    }
    return allowedOverridesDateFormat.includes(forceDateFormat) && forceDateFormat
}

const useConfigDateFormatOverride = getDateFormatOverride();
```